RecoEcal/EgammaClusterProducers/python/SCEnergyCorrectorDRNProducer_cfi.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+DRNProducerEB = cms.EDProducer('SCEnergyCorrectorDRNProducer',
+   correctorCfg = cms.PSet(
+     ecalRecHitsEE = cms.InputTag('reducedEcalRecHitsEE'),
+     ecalRecHitsEB = cms.InputTag('reducedEcalRecHitsEB'),
+     rhoFastJet = cms.InputTag("fixedGridRhoAll"),
+   ),
+   inputSCs = cms.InputTag('particleFlowSuperClusterECAL','particleFlowSuperClusterECALBarrel'),
+   Client = cms.PSet(
+       mode = cms.string("Async"),
+        preferredServer = cms.untracked.string(""),
+        timeout = cms.untracked.uint32(10),
```

Comment:
Should these come as defaults from a Triton config? I suspect having timeouts and sync/async configurations scattered among physics configurations might not be ideal.

Reply:
The appropriate timeout value is algorithm-dependent.

Reply:
> This seems like a short timeout. Are you sure the actual inference will never take this long? (The point of the timeout is to identify server-side issues, rather than waiting for a request that would never return because the server is not operating properly.)

I definitely have never seen a crash due to this short timeout. What is the best way to robustly determine what an appropriate value for this is? Also, what are its units? msec?

Reply:
The timeout is in seconds. The only good way to determine it is to run a bunch of inference requests and look at the average and maximum time, then inflate it by some amount in case of e.g. higher network latency.

Reply:
Okay, cool. Seeing as the design use case here is the local fallback server, I am inclined to think that a 10 second timeout is more than enough. If it takes anywhere on the order of seconds for a local inference request to go through I think there is definitely a problem (we're anticipating an inference duration on the order of 10 msec per event).